english/order-pizza/order-pizza.md

```diff
@@ -1,3 +1,10 @@
-Your hungry and tired, pizza is the only thing that will cheer you up.
-But you dont know what to order.
-Suddenly a [menu](https://www.pizzahut.com.au/) appears!
\ No newline at end of file
+Your hungry and tired, pizza is the only thing that will cheer you up. You
+thought long and hard about fried chicken, or perhaps all you can eat mongolian
+stir fry but in the end the only thing that truly matters to a programmer is hot delivery pizza. When it's done you'll hang it from your window in a pillowcase
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.